--- origsrc/kmymoney-3.95.0/CMakeLists.txt	2009-12-28 05:21:22.000000000 -0600
+++ src/kmymoney-3.95.0/CMakeLists.txt	2010-01-13 19:28:50.305639000 -0600
@@ -234,7 +234,7 @@ ENDIF(CMAKE_SYSTEM_NAME MATCHES Linux)
 
 IF(NOT WIN32)
   # use position independent code
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CMAKE_SHARED_LIBRARY_CXX_FLAGS}")
 ENDIF(NOT WIN32)
 IF(NOT MSVC)
   # be more pedantic about common symbols
--- origsrc/kmymoney-3.95.0/kmymoney/plugins/printcheck/numbertowords.h	2009-12-28 05:16:32.000000000 -0600
+++ src/kmymoney-3.95.0/kmymoney/plugins/printcheck/numbertowords.h	2010-01-13 20:05:07.181149200 -0600
@@ -22,7 +22,7 @@
 
 #include <qstringlist.h>
 
-#include <kmymoney/mymoneymoney.h>
+#include <mymoneymoney.h>
 
 class MyMoneyMoneyToWordsConverter
 {
--- origsrc/kmymoney-3.95.0/kmymoney/plugins/printcheck/printcheck.cpp	2009-12-28 05:16:32.000000000 -0600
+++ src/kmymoney-3.95.0/kmymoney/plugins/printcheck/printcheck.cpp	2010-01-13 20:18:25.656819400 -0600
@@ -31,8 +31,8 @@
 #include <kplugininfo.h>
 
 // KMyMoney includes
-#include <kmymoney/mymoneyfile.h>
-#include <kmymoney/pluginloader.h>
+#include "mymoneyfile.h"
+#include "pluginloader.h"
 
 #include "numbertowords.h"
 #include "pluginsettings.h"
--- origsrc/kmymoney-3.95.0/kmymoney/plugins/printcheck/printcheck.h	2009-12-28 05:16:32.000000000 -0600
+++ src/kmymoney-3.95.0/kmymoney/plugins/printcheck/printcheck.h	2010-01-13 20:17:46.615586300 -0600
@@ -20,10 +20,10 @@
 #ifndef __PRINTCHECK_H__
 #define __PRINTCHECK_H__
 
-#include <kmymoney/kmymoneyplugin.h>
-#include <kmymoney/mymoneyaccount.h>
-#include <kmymoney/mymoneykeyvaluecontainer.h>
-#include <kmymoney/selectedtransaction.h>
+#include "kmymoneyplugin.h"
+#include "mymoneyaccount.h"
+#include "mymoneykeyvaluecontainer.h"
+#include "selectedtransaction.h"
 
 class QStringList;
 class KPluginInfo;
--- origsrc/kmymoney-3.95.0/kmymoney/views/kbudgetviewdecl.ui	2009-12-28 05:16:50.000000000 -0600
+++ src/kmymoney-3.95.0/kmymoney/views/kbudgetviewdecl.ui	2010-01-13 20:12:01.238831900 -0600
@@ -269,7 +269,7 @@
   <customwidget>
    <class>KBudgetValues</class>
    <extends>QWidget</extends>
-   <header>kmymoney/kbudgetvalues.h</header>
+   <header>kmymoney/widgets/kbudgetvalues.h</header>
   </customwidget>
   <customwidget>
    <class>Q3Frame</class>
--- origsrc/kmymoney-3.95.0/kmymoney/views/kpayeesviewdecl.ui	2009-12-28 05:16:50.000000000 -0600
+++ src/kmymoney-3.95.0/kmymoney/views/kpayeesviewdecl.ui	2010-01-13 20:13:41.143546100 -0600
@@ -534,7 +534,7 @@ new transactions with this payee</string
   <customwidget>
    <class>KMyMoneyAccountCombo</class>
    <extends>KComboBox</extends>
-   <header>kmymoney/kmymoneyaccountcombo.h</header>
+   <header>kmymoney/widgets/kmymoneyaccountcombo.h</header>
   </customwidget>
   <customwidget>
    <class>Q3GroupBox</class>
