--- origsrc/kmymoney-3.95.0/libkdchart/src/KDChartPercentPlotter_p.cpp	2009-12-28 05:16:26.000000000 -0600
+++ src/kmymoney-3.95.0/libkdchart/src/KDChartPercentPlotter_p.cpp	2010-01-14 01:08:39.400828500 -0600
@@ -183,12 +183,12 @@ void PercentPlotter::paint( PaintContext
 
                 data.first = leftY + ( rightY - leftY ) * ( xValue - leftX ) / ( rightX - leftX );
                 // if the result is a valid value, let's assign the index, too
-                if( !ISNAN( data.first ) )
+                if( !ISNAN( double(data.first) ) )
                     data.second = left.second;
             }
         
             // sum it up
-            if( !ISNAN( yValues[ column ].first ) )
+            if( !ISNAN( double(yValues[ column ].first) ) )
                 yValueSums[ xValue ] += yValues[ column ].first;
         }
     }
