--- origsrc/kmymoney-3.96.1/libkdchart/src/KDChartPercentPlotter_p.cpp	2010-02-15 04:48:10.000000000 -0600
+++ src/kmymoney-3.96.1/libkdchart/src/KDChartPercentPlotter_p.cpp	2010-02-18 00:19:08.549945200 -0600
@@ -185,12 +185,12 @@ void PercentPlotter::paint( PaintContext
 
                 data.first = leftY + ( rightY - leftY ) * ( xValue - leftX ) / ( rightX - leftX );
                 // if the result is a valid value, let's assign the index, too
-                if( !ISNAN( data.first ) )
+                if( !ISNAN( double(data.first) ) )
                     data.second = left.second;
             }
 
             // sum it up
-            if( !ISNAN( yValues[ column ].first ) )
+            if( !ISNAN( double(yValues[ column ].first) ) )
                 yValueSums[ xValue ] += yValues[ column ].first;
         }
     }
