--- origsrc/kmymoney-3.97.2/kmymoney/plugins/kbanking/CMakeLists.txt	2010-04-06 14:04:39.000000000 -0500
+++ src/kmymoney-3.97.2/kmymoney/plugins/kbanking/CMakeLists.txt	2010-04-29 20:54:47.911785400 -0500
@@ -22,11 +22,11 @@ SET(kmm_kbanking_PART_SRCS kbanking.cpp
 KDE4_ADD_PLUGIN(kmm_kbanking ${kmm_kbanking_PART_SRCS})
 
 TARGET_LINK_LIBRARIES(kmm_kbanking
-                      kmm_mymoney
                       kmm_kbanking_dialogs
                       kmm_kbanking_views
                       kmm_kbanking_widgets
                       kmm_widgets
+                      kmm_mymoney
                       kmm_plugin
                       ${KDE4_KDEUI_LIBS}
                       ${KDE4_KDE3SUPPORT_LIBS}
