--- origsrc/kmymoney-4.5.2/kmymoney/mymoney/mymoneyfinancialcalculator.h	2010-12-23 03:21:47.000000000 -0600
+++ src/kmymoney-4.5.2/kmymoney/mymoney/mymoneyfinancialcalculator.h	2011-01-14 09:48:13.377398800 -0600
@@ -40,7 +40,7 @@
 #include <kmm_mymoney_export.h>
 
 
-#ifdef SIZEOF_LONG_DOUBLE
+#if defined(SIZEOF_LONG_DOUBLE) && !defined(__CYGWIN__)
 #define FCALC_DOUBLE  long double
 #else
 #define FCALC_DOUBLE  double
--- origsrc/kmymoney-4.5.2/kmymoney/wizards/newloanwizard/durationwizardpage.cpp	2010-12-23 03:21:34.000000000 -0600
+++ src/kmymoney-4.5.2/kmymoney/wizards/newloanwizard/durationwizardpage.cpp	2011-01-14 10:04:31.400597600 -0600
@@ -85,7 +85,7 @@ int DurationWizardPage::term(void) const
   return factor;
 }
 
-QString DurationWizardPage::updateTermWidgets(const long double val)
+QString DurationWizardPage::updateTermWidgets(const FCALC_DOUBLE val)
 {
   long long vl = static_cast<long long>(floorl(val));
 
--- origsrc/kmymoney-4.5.2/kmymoney/wizards/newloanwizard/durationwizardpage.h	2010-12-23 03:21:34.000000000 -0600
+++ src/kmymoney-4.5.2/kmymoney/wizards/newloanwizard/durationwizardpage.h	2011-01-14 10:05:06.609859000 -0600
@@ -29,6 +29,7 @@
 // ----------------------------------------------------------------------------
 // Project Includes
 
+#include "mymoneyfinancialcalculator.h"
 #include "ui_durationwizardpagedecl.h"
 
 /**
@@ -49,7 +50,7 @@ class DurationWizardPage : public Durati
 public:
   explicit DurationWizardPage(QWidget *parent = 0);
 
-  QString updateTermWidgets(const long double val);
+  QString updateTermWidgets(const FCALC_DOUBLE val);
   int term(void) const;
 
 public slots:
