--- origsrc/kmymoney-4.7.2/CMakeLists.txt	2015-04-25 12:20:16.000000000 -0500
+++ src/kmymoney-4.7.2/CMakeLists.txt	2015-05-13 19:18:46.706999100 -0500
@@ -234,7 +234,7 @@ endif(CMAKE_SYSTEM_NAME MATCHES Linux)
 
 if(NOT WIN32)
   # use position independent code
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CMAKE_SHARED_LIBRARY_CXX_FLAGS}")
 endif(NOT WIN32)
 if(CMAKE_COMPILER_IS_GNUCXX)
   # be more pedantic about common symbols
