--- origsrc/kmymoney-4.8.0/kmymoney/mymoney/mymoneyaccount.h	2016-06-14 14:16:15.573035100 -0500
+++ src/kmymoney-4.8.0/kmymoney/mymoney/mymoneyaccount.h	2016-08-29 15:56:25.643582600 -0500
@@ -770,8 +770,10 @@ QList< payeeIdentifierTyped<type> > MyMo
   return typedList;
 }
 
+#if 0
 template<>
 QList< payeeIdentifierTyped< ::payeeIdentifiers::ibanBic> > MyMoneyAccount::payeeIdentifiersByType() const;
+#endif
 
 /**
   * Make it possible to hold @ref MyMoneyAccount objects inside @ref QVariant objects.
--- origsrc/kmymoney-4.8.0/kmymoney/mymoney/storage/CMakeLists.txt	2016-06-14 14:16:15.582035100 -0500
+++ src/kmymoney-4.8.0/kmymoney/mymoney/storage/CMakeLists.txt	2016-08-28 23:05:03.573097600 -0500
@@ -18,6 +18,7 @@ set(storage_a_SOURCES
 # This library is actually not needed. It is built just for
 # convenience, and then linked into kmm_mymoney.
 kde4_add_library( kmm_storage STATIC ${storage_a_SOURCES} )
+set_target_properties(kmm_storage PROPERTIES COMPILE_DEFINITIONS "MAKE_KMM_MYMONEY_LIB")
 
 ########### install files ###############
 
